packages/manager-components/src/components/ManagerText/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,5 @@
+{
+  "iam_hidden_text": "Hidden",
```

Comment:
**nitpick:** We need it in French.

Reply:
I'm in waiting for the trad in french ❓ we dont know what can be the good translation